runtime/moonbase/tests/common/mod.rs

```diff
@@ -49,6 +49,17 @@ pub fn last_event() -> Event {
 	System::events().pop().expect("Event expected").event
 }
 
+// Helper function to give a simple evm context suitable for tests.
+// this should probably go somewhere else. Maybe we should give it a
+// impl Default for Context upstream.
```

Comment:
> Maybe we should give it a impl Default for Context upstream.

This would make sense. It is a useful helper function, used often in the code.

Reply:
https://github.com/rust-blockchain/evm/pull/35